### DISCUSSION TITLE
Nicer README

### DISCUSSION THREAD

**Discussion by irevenko** [sammy-ette/Hilbish#31] [Ideas]
Since project is growing you should add 
- Screenshots section
- Usage section
- Contributions section

**Comment by sammy-ette** [comment]
I get what the screenshots and contributions sections are, but what will usage be for?

**Reply by devins2518** [reply]
I think once plugins get added and supported, it could be to show off how to get started with petals. But CONTIBUTE.md is definitely needed right now, 2 prs that I know of failed the semantic check.

**Reply by irevenko** [reply]
yes

**Comment by sammy-ette** [comment]
done!! usage section will come when petals are added
